protocol/alterclientquotas/alterclientquotas.go

```diff
@@ -0,0 +1,50 @@
+package alterclientquotas
+
+import "github.com/segmentio/kafka-go/protocol"
+
+func init() {
+	protocol.Register(&Request{}, &Response{})
+}
+
+// Detailed API definition: https://kafka.apache.org/protocol#The_Messages_AlterClientQuotas
+type Request struct {
```

Comment:
I believe these also need a tagged field to indicated v1 is flexible. It looks like the string fields also became compact in v1.